booking-engine-api/operations/availability-blocks.md

```diff
@@ -0,0 +1,66 @@
+# Availability blocks
+
+## Get Availability blocks
+
+Get availability block details for specified booking engine instance, that modifies your availability of space categories, and rates.
+This operation can be called initially to fetch data which may be important during the booking workflow.
+Availability block can restrict your booking engine's calendar to specific interval defined by `StartUtc` and `EndUtc` in the response, it also gives you `RateId` that should be used.
+
+### Request
+
+`[ApiBaseUrl]/api/distributor/v1/availabilityBlocks/getAll`
+
+```json
+{
+    "Client": "My Client 1.0.0",
+    "AvailabilityBlockIds": [
+        "5mgbe1b4-6739-40b7-81b3-d369d9469c48"
+    ],
+    "EnterpriseId": "3edbe1b4-6739-40b7-81b3-d369d9469c48"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Client` | string | required | Identification of the API client, as described in [Authorization](../guidelines/authorization.md). |
+| `EnterpriseId` | string | required | Unique identifier of the hotel (enterprise). |
+| `AvailabilityBlockIds` | array of string | required | Set of unique identifier of availabilityBlockId usually provided in query parameter on private link to your booking engine. |
```

Comment:
I would change this to "Set of unique identifiers of the Availability Blocks for which you want to get the details"